packages/basis/src/loader/BasisParser.ts

```diff
@@ -316,6 +316,7 @@ export class BasisParser
                 pvrtc: gl.getExtension('WEBGL_compressed_texture_pvrtc')
                     || gl.getExtension('WEBKIT_WEBGL_compressed_texture_pvrtc'),
                 atc: gl.getExtension('WEBGL_compressed_texture_atc'),
+                bptc: gl.getExtension('EXT_texture_compression_bptc'),
```

Comment:
Given that it provides one of the highest texture qualities, I believe it's better to declare 
`bptc: gl.getExtension('EXT_texture_compression_bptc'),` as high as possible.

Reply:
I've created a separate issue for that. 
https://github.com/pixijs/pixijs/issues/10213